distribution/docs/src/main/resources/_contents/_security-framework/security-ldap-embedded-server-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+==== Embedded LDAP Server
+
+${branding} includes an embedded LDAP server (OpenDJ) for testing and demonstration purposes
+
+===== Installing the Embedded LDAP Server
+
+The embedded LDAP server is not installed by default with a standard installation.
+
+. Navigate to the ${admin-console}.
+. Select *Manage* Applications.
+. Install the *OpenDJ Embedded* application.
+
+===== Configuring the Embedded LDAP
+
+The configuration options are located on the ${admin-console} under *Opendj Embedded -> Configuration -> LDAP Server*. It currently contains three configuration options.
+
+The configuration options are located on the standard ${branding} configuration ${admin-console} under the title *LDAP Server*.
+It currently contains three configuration options.
+
+.OpenDJ Embedded Configurable Properties
+[cols="1,7" options="header"]
+|===
+
+|Configuration Name
+|Description
+
+|LDAP Port
+|Sets the port for LDAP (plaintext and startTLS). 0 will disable the port.
+
+|LDAPS Port
+|Sets the port for LDAPS. 0 will disable the port.
+
+|Base LDIF File
+|Location on the server for a LDIF file.
+This file will be loaded into the LDAP and overwrite any existing entries.
+This option should be used when updating the default groups/users with a new LDIF file for testing.
+The LDIF file being loaded may contain any LDAP entries (schemas, users, groups, etc.).
+If the location is left blank, the default base LDIF file will be used that comes with ${branding}.
+
+|===
+
+===== Trust Certificates
+
+For LDAPS or startTLS to function correctly, it is important that the LDAP server is configured with a keystore file that trusts the clients it is connecting to and vice versa.
+Complete the following procedure to provide your own keystore information for the LDAP. 
+
+. The embedded LDAP server is not recommended for production use as it has hardcoded keystores and truststores with localhost certificates. These cannot be changed unless the embedded server bundle is re-built with new stores.
```

Comment:
This isn't true anymore about the hardcoded keystore/truststore. However, I still don't want to advertise it as anything other than test use.